app/controllers/events/view/tickets/orders/list.js

```diff
@@ -74,6 +74,26 @@ export default Controller.extend({
         .finally(() => {
           this.set('isLoading', false);
         });
+    },
+    async resendConfirmation(order) {
+      let payload = {};
+      try {
+        payload = {
+          'data': {
+            'order' : order.identifier,
+            'user'  : this.authManager.currentUser.email
+          }
+        };
+        await this.loader.post('orders/resend-email', payload);
+        this.notify.success(this.l10n.t('Email confirmation has been sent to attendees successfully'));
+      } catch (error) {
+        if (error.message === 'TOO MANY REQUESTS') {
```

Comment:
@niranjan94 @shreyanshdwivedi @CosmicCoder96 Will this workaround be okay?

Reply:
@uds5501 this is very wrong. You must use the status code.

Modify the loader service to ensure the original response is sent as the error along with the message that is being currently sent there

https://github.com/fossasia/open-event-frontend/blob/development/app/services/loader.js#L100

Reply:
@niranjan94 Understood.